app/routes/surveys/components/Submissions/SubmissionPublicResults.js

```diff
@@ -28,15 +28,13 @@ const SubmissionPublicResultsPage = ({
 
   const generateTextAnswers = (question) => {
     let texts = [];
-    Object.keys(results[question.id])
-      .map((name) => {
-        if (name !== 'questionType') {
-          texts = results[question.id][name].map((answer, i) => (
-            <li key={i}>{answer}</li>
-          ));
-        }
-      })
-      .filter(Boolean);
+    Object.keys(results[question.id]).forEach((name) => {
```

Comment:
Hahaha, wut, I don't really understand what this func is doing.. :sweat_smile: But this doesn't change the logic, so I guess it's ok!

Reply:
It creates a list of ```<li>``` elements, but the function is a bit obscure. The .filter() did nothing, since we don't use the return type, so I just removed it. :shrug: